data/json/terrain.json

```diff
@@ -1051,6 +1107,7 @@
             "t_wall_metal_v"
         ],
         "name": "metal wall",
+        "description": "An industrially fabricated thick sheet of carefully positioned and joined seamlessly with perimeter sealant, this wall is capable of resisting extreme elements as well as hostile forces.  Blast load rated and extremely fire-retardant, any breaches will require specialized tools or industrial vehicles.",
```

Comment:
> ...[a] thick sheet of carefully positioned and joined seamlessly with perimeter sealant

What's carefully positioned?

> any breaches will require specialized tools or industrial vehicles

Consider "breaching will require," since it's the act of breaching that will require those items